package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "start": "node index",
     "prod-start": "NODE_ENV=production APP_ENV=production node index",
-    "dev-api-start": "APP_ENV=development node index",
-    "prod-api-start": "APP_ENV=production node index",
+    "dev-api-start": "APP_ENV=development KMS_ENV=unencrypted node index",
```

Comment:
I understand we want to make a request to `APP_ENV=production` with encrypted keys all the time. But does this mean we can't run `APP_ENV=development` with encrypted key locally?